tee-worker/omni-executor/intent/executors/cross-chain/src/cross_chain.rs

```diff
@@ -15,8 +15,10 @@ impl<BinanceClient: BinanceApi, SolanaClient: SolanaClientTrait>
 		swap_order: &SwapOrder,
 		pumpx_config: &PumpxConfig,
 	) -> Result<String, ()> {
-		// payout_amount
-		debug!("executing cross chain swap");
```

Comment:
This should be already logged

```
2025-05-20T15:26:14.924331Z  INFO executor_core::native_task: Assigning id: e1deb490-f5f8-4fb7-a720-7decead74688 to task: RequestIntent(Substrate(Address32), 1, CallEthereum(CallEthereum { address: 0x0000000000000000000000000000000000000000, input: BoundedVec([], 10240) }))
2025-05-20T15:26:15.086868Z  INFO native-task{id="e1deb490-f5f8-4fb7-a720-7decead74688"}: parentchain_signer::tx_signer: Signing call OmniAccount::intent_accepted with nonce 0
```

Maybe we need to adjust first entry  ?

Reply:
> This should be already logged
> 
> ```
> 2025-05-20T15:26:14.924331Z  INFO executor_core::native_task: Assigning id: e1deb490-f5f8-4fb7-a720-7decead74688 to task: RequestIntent(Substrate(Address32), 1, CallEthereum(CallEthereum { address: 0x0000000000000000000000000000000000000000, input: BoundedVec([], 10240) }))
> 2025-05-20T15:26:15.086868Z  INFO native-task{id="e1deb490-f5f8-4fb7-a720-7decead74688"}: parentchain_signer::tx_signer: Signing call OmniAccount::intent_accepted with nonce 0
> ```
> 
> Maybe we need to adjust first entry ?

Which exactly? the intent_id or omni-account, or both?